axelar-chains-config/tests/schema/index.js

```diff
@@ -5,7 +5,7 @@ const axelarSchema = {
         id: { type: 'string' },
         axelarId: { type: 'string' },
         rpc: { type: 'string' },
-        lcd: { type: 'string' },
+        lcd: { type: 'string', pattern: '^$|^(https?:\\/\\/[^\\/\\:]+(:\\d+)?)$' },
```

Comment:
could be `http` (e.g. devnets). regex is not very readable, add a comment with the example format, or don't keep it too restrictive

Reply:
ah, missed the `?`